shell/gpu/gpu_surface_gl.h

```diff
@@ -20,6 +20,8 @@ namespace flutter {
 
 class GPUSurfaceGL : public Surface {
  public:
+  static sk_sp<GrDirectContext> MakeGLContext(GPUSurfaceGLDelegate* delegate);
```

Comment:
If possible, can we add some consistency between the metal version and this one? In metal, the IOSContextMetal just decides to make a GrDirectContext the way it wants without asking anyone. Here, it seems like the IOSSurfaceGL asks the GPUSurfaceGL to make one, then pushes it into IOSContextGL which then owns it.